Makefile.am

```diff
@@ -69,6 +70,7 @@ all-local:
 python_prefix = --prefix=$(prefix)
 
 install-exec-local:
+	test -d $(var_log_crmsh) || $(INSTALL) -d -m 770 $(var_log_crmsh)
```

Comment:
The previous solution was good enough. We don't need to do the test. We can just install the directory.